uploadNeuroDB/NeuroDB/MRIProcessingUtility.pm

```diff
@@ -695,9 +790,13 @@ sub dicom_to_minc {
         close MAIL;
         croak("dicom_to_minc failure, exit code $exit_code");
```

Comment:
@nicolasbrossard ,
should we spool there? it is an error at this point but it does NOT get logged... maybe we should spool anyway?